src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/LanguageServices/Workspace.cs

```diff
@@ -525,16 +525,14 @@ async Task ApplyInBatchAsync()
                     isActiveEditorContext: _activeEditorContextTracker.IsActiveEditorContext(ContextId),
                     isActiveConfiguration: IsPrimary);
 
-                Context.StartBatch();
-
                 try
                 {
+                    await using IAsyncDisposable _ = await Context.CreateBatchScopeAsync(cancellationToken);
+
                     applyFunc(update, contextState, cancellationToken);
                 }
                 finally
```

Comment:
Do we need to log here incase of an exception or the messages are bubbled in the stack trace?